guides/routing.md

```diff
@@ -691,7 +691,7 @@ Endpoints organize all the plugs common to every request, and apply them before
 
 - [Plug.RequestId](https://hexdocs.pm/plug/Plug.RequestId.html) - generates a unique request id for each request.
 
-- [Plug.Logger](https://hexdocs.pm/plug/Plug.Logger.html) - logs incoming requests
+- [Plug.Telemetry](https://hexdocs.pm/plug/Plug.Telemetry.html) - adds instrumentation points so Phoenix can log the srequest path, status code and request time by default.
```

Comment:
"srequest" should be "request"

Reply:
Woops, sorry about that.. fixed!

Should I squash the changes?